### PR TITLE
Add awesome prints

### PR DESCRIPTION
**Pull request by 552020**
I added in this MR all  the prints I used to spot the error in main. Enjoy!

### CODE REVIEW

**Review by lmangall — DISMISSED** [review]
LGTM!

**Comment by 552020** [comment]
What is this the merge-base changed after approval?

**Comment by 552020** [comment]
Do you think that this is just about the fact that some other PR got merged and then the merge base changed and even it is not conflicting with this one the reviews are considered stale. I don't think we had this before ... I mean is a common practice to approve and merge multiple PR ...

**Comment by lmangall** [comment]
I am not sure, it's quite annoying

File: src/Server.hpp
Comment (lmangall) [inline src/Server.hpp]: What is this change ?

**Review by dantol29 — DISMISSED** [review]
Looks good!

But can you explain acceptNewConnection changes?

File: src/server_utils.hpp
Comment (lmangall) [inline src/server_utils.hpp:37]: We have quite a few prints set by overloading the << operator, wouldn't that be applicable here ?

**Comment by lmangall** [comment]
> Sure! What is not clear?

I added an inline comment on the code about it

**Comment by 552020** [comment]
> > Sure! What is not clear?
> 
> I added an inline comment on the code about it

Normally you are supposed to see in the main page of the pull request but I can't see it.

**Comment by lmangall** [comment]
@552020 Sorry, It's because I had not finished the review,

File: src/Server.hpp
Comment (552020) [reply src/Server.hpp]: Ah. Before we were taking the _serverFd (as we had only one). This is the preparation for the multi 'server' thing. We are taking the Server Connection Object which has the POLLIN, to pass the information of this server connection to the client connection we are going to create. Maybe it would have been better to call the parameter Connection &ServerConn

**Comment by 552020** [comment]
Before 

image


After

image


I thought I changed the message of the only commit in something more meaningful but this didn't work

File: src/Server.hpp
Comment (lmangall) [reply src/Server.hpp]: Alright, thanks for the explanation

**Review by lmangall — APPROVED** [review]
LGTM!

**Review by dantol29 — APPROVED** [review]
I approve after correct merge rebase

**Pull request merged**
